network/player.go

```diff
@@ -23,23 +35,44 @@ func ConvertToGetPlayerResponse(p model.Player) GetPlayerResponse {
 			ID:   p.ID,
 			Name: p.Name,
 		},
-		Games: convertGames(p.Games),
+		Games: convertToParticipatingGames(p.Games),
 	}
 }
 
-func convertGames(modelGames map[model.GameID]model.PlayerColor) map[model.GameID]string {
+func convertToParticipatingGames(modelGames map[model.GameID]model.PlayerColor) map[model.GameID]string {
```

Comment:
:cactus: 📖 `modelGames` -> `mgs`? (I know I was the one who originally named the var... I'm getting more accustomed to golang naming conventions now :#)